style.css

```diff
@@ -1,51 +1,279 @@
+@keyframes move-twink-back {
+    from {background-position:0 0;}
+    to {background-position:-10000px 5000px;}
+}
+
+@keyframes move-clouds-back {
+    from {background-position:0 0;}
+    to {background-position:10000px 0;}
+}
+
+@keyframes glitch {
+    0% {
+        clip-path: inset(71% 0 10% 0);
+        transform: translate(-5px, 5px);
+    }
+    5% {
+        clip-path: inset(29% 0 71% 0);
+        transform: translate(5px, -5px);
+    }
+    10% {
+        clip-path: inset(83% 0 1% 0);
+        transform: translate(-5px, 5px);
+    }
+    15% {
+        clip-path: inset(100% 0 1% 0);
+        transform: translate(5px, -5px);
+    }
+    20% {
+        clip-path: inset(58% 0 43% 0);
+        transform: translate(-5px, 5px);
+    }
+    25% {
+        clip-path: inset(46% 0 34% 0);
+        transform: translate(5px, -5px);
+    }
+    100% {
+        clip-path: inset(100% 0 0% 0);
+        transform: translate(0);
+    }
+}
+
+@keyframes rotate-sun {
+    0% {
+        transform: rotate(0deg);
+    }
+    100% {
+        transform: rotate(360deg);
+    }
+}
+
+@keyframes pulse-glow {
+    0%, 100% {
+        box-shadow: 0 0 50px 20px rgba(255, 153, 51, 0.8);
+    }
+    50% {
+        box-shadow: 0 0 80px 40px rgba(255, 153, 51, 0.6);
+    }
+}
+
 html, body {
-    background-color: rgb(122, 194, 126);
-    background-image: url("img/sun.png");
-    background-repeat: no-repeat;
     margin: 0;
     padding: 0;
+    font-family: 'Roboto', sans-serif;
+    color: #ffffff;
+    background: linear-gradient(to bottom, #0f2027, #203a43, #2c5364);
+    overflow-x: hidden;
+}
+
+.sun-container {
+    position: absolute;
+    top: -100px;
+    left: -100px;
+    width: 300px;
+    height: 300px;
+    z-index: -4;
+    animation: pulse-glow 10s infinite ease-in-out;
+    border-radius: 50%;
+}
+
+.sun {
+    width: 100%;
+    height: 100%;
+    animation: rotate-sun 60s linear infinite;
+}
+
+.stars, .twinkling, .clouds {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: 100%;
+    height: 100%;
+    display: block;
+}
+
+.stars {
+    background: #000 url(http://www.script-tutorials.com/demos/360/images/stars.png) repeat top center;
+    z-index: -3;
+}
+
+.twinkling {
+    background: transparent url(http://www.script-tutorials.com/demos/360/images/twinkling.png) repeat top center;
+    z-index: -2;
+    animation: move-twink-back 200s linear infinite;
+}
+
+.clouds {
+    background: transparent url(http://www.script-tutorials.com/demos/360/images/clouds3.png) repeat top center;
```

Comment:
Same here. If you need a image put it in the img folder. Also that does not look like a teeworlds image.